components/dashboard/src/teams/TeamPolicies.tsx

```diff
@@ -121,7 +122,7 @@ export default function TeamPoliciesPage() {
         [workspaceTimeout, allowTimeoutChangeByMembers, handleUpdateTeamSettings],
     );
 
-    const billingModeAllowsWorkspaceTimeouts =
+    const isPaidPlan =
```

Comment:
What about this, as `isPaidPlan` is a bit misleading in the Dedicated case (`billingMode === "none"`):
```suggestion
    const isWorkspaceTimeoutsAllowed =
```